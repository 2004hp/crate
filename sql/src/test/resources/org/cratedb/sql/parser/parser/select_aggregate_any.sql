select any(col1) from table1 group by col2 having any(col1) > 0 order by any(col1)