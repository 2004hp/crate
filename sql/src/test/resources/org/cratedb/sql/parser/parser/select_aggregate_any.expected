org.cratedb.sql.parser.parser.CursorNode@517c555d
name: null
updateMode: UNSPECIFIED
statementType: SELECT
resultSet: 	
	org.cratedb.sql.parser.parser.SelectNode@39c87676
	isDistinct: false
	resultColumns: 		
		org.cratedb.sql.parser.parser.ResultColumnList@17d95a9d

		[0]:		
		org.cratedb.sql.parser.parser.ResultColumn@3ecc05a6
		exposedName: null
		name: null
		tableName: null
		isDefaultColumn: false
		type: null
		expression: 			
			org.cratedb.sql.parser.parser.AggregateNode@4486f57f
			aggregateName: ANY
			operator: null
			methodName: null
			type: null
			operand: 				
				org.cratedb.sql.parser.parser.ColumnReference@785f87fe
				columnName: col1
				tableName: null
				type: null
	fromList: 		
		org.cratedb.sql.parser.parser.FromList@47d9726f

		[0]:		
		org.cratedb.sql.parser.parser.FromBaseTable@14e7fa43
		tableName: table1
		updateOrDelete: null
		null
		correlation Name: null
		null
	groupByList:		
		org.cratedb.sql.parser.parser.GroupByList@9df4f6e

		[0]:		
		org.cratedb.sql.parser.parser.GroupByColumn@23d7c3d2
		columnPosition: -1
		columnExpression: 			
			org.cratedb.sql.parser.parser.ColumnReference@2a792260
			columnName: col2
			tableName: null
			type: null
	havingClause:		
		org.cratedb.sql.parser.parser.BinaryRelationalOperatorNode@76bfe268
		operator: >
		methodName: greaterThan
		type: null
		leftOperand: 			
			org.cratedb.sql.parser.parser.AggregateNode@2fa956c0
			aggregateName: ANY
			operator: null
			methodName: null
			type: null
			operand: 				
				org.cratedb.sql.parser.parser.ColumnReference@167e6fb2
				columnName: col1
				tableName: null
				type: null
		rightOperand: 			
			org.cratedb.sql.parser.parser.NumericConstantNode@46d71aa7
			value: 0
			type: INTEGER NOT NULL
orderByList: 	
	org.cratedb.sql.parser.parser.OrderByList@5416ee46
	allAscending: true
	[0]:	
	org.cratedb.sql.parser.parser.OrderByColumn@2d7560cb
	columnPosition: -1
	expression: 		
		org.cratedb.sql.parser.parser.AggregateNode@7525c6bd
		aggregateName: ANY
		operator: null
		methodName: null
		type: null
		operand: 			
			org.cratedb.sql.parser.parser.ColumnReference@31ade61
			columnName: col1
			tableName: null
			type: null
